composer.json

```diff
@@ -17,7 +17,8 @@
     },
     "require": {
         "psr/log": "~1.0",
-        "php" : "~5.4"
+        "php" : "~5.4",
+        "container-interop/container-interop": "^1.1"
```

Comment:
This is a pretty specialized adapter, please move the requirement to "require-dev". 
Projects which use container-interpop will have it as its own dependency already.